chapters/4-acquisition.tex

```diff
@@ -499,6 +524,8 @@ \subsection{Using electronic survey features}
 
 The final stage of survey piloting, the data-focused pilot, should be done at this stage (after the questionnaire is programmed).
 The objective of this \textbf{data-focused pilot}\sidenote{
+  A checklist with best practices important to remember during a data-focused pilot
```

Comment:
Note that @Avnish95 is changing the name of this page, need to update link once that's done

Reply:
done in 04a6a926ec8986149a58e7f5e0f4dc8848a1dc3a